new-docs/source/tutorial/1-getting-started.ipynb

```diff
@@ -0,0 +1,340 @@
+    "Next we create a directory containing 10 randomly generated [NIfTI](https://nifti.nimh.nih.gov/) files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from fileformats.medimage import Nifti1\n",
+    "\n",
+    "nifti_dir = test_dir / \"nifti\"\n",
+    "nifti_dir.mkdir()\n",
+    "\n",
+    "for i in range(10):\n",
+    "    Nifti1.sample(nifti_dir, seed=i)  # Create a dummy NIfTI file in the dest. directory"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that when you run concurrent processes within a Jupyter notebook the following snippet\n",
+    "is also required"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import nest_asyncio\n",
+    "nest_asyncio.apply()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Running your first task\n",
+    "\n",
+    "Pre-defined task definitions are installed under the `pydra.tasks.*` namespace by separate\n",
+    "task packages (e.g. `pydra-fsl`, `pydra-ants`, ...). To use a pre-defined task definition\n",
+    "\n",
+    "* import the class from the `pydra.tasks.*` package it is in\n",
+    "* instantiate it with appropriate parameters\n",
+    "* \"call\" resulting object (i.e. `my_task(...)`) to execute it as you would a function \n",
+    "\n",
+    "To demonstrate with an example of loading a JSON file with the\n",
+    "`pydra.tasks.common.LoadJson` task, we first create an example JSON file to test with"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can load the JSON contents back from the file using the `LoadJson` task definition\n",
+    "class"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import the task definition\n",
+    "from pydra.tasks.common import LoadJson\n",
+    "\n",
+    "# Instantiate the task definition, providing the JSON file we want to load\n",
+    "load_json = LoadJson(file=json_file)\n",
+    "\n",
+    "# Run the task to load the JSON file\n",
+    "outputs = load_json()\n",
+    "\n",
+    "# Access the loaded JSON output contents and check they match original\n",
+    "assert outputs.out == JSON_CONTENTS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Iterating over inputs\n",
+    "\n",
+    "It is straightforward to apply the same operation over a set of inputs using the `split()`\n",
+    "method. For example, if we wanted to re-grid all the NIfTI images stored in a directory,\n",
+    "such as the sample ones generated by the code below"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we can by importing the `MrGrid` shell-command task from the `pydra-mrtrix3` package\n",
```

Comment:
```suggestion
    "such as the sample ones generated by `Nifti1` before, we can import the `MrGrid` shell-command task from the `pydra-mrtrix3` package\n",
```

Reply:
I think this suggestion deletes some preceding lines unintentionally